content/en/blog/_posts/2025-01-22-kube-scheduler-simulator/index.md

```diff
@@ -0,0 +1,138 @@
+    kube-scheduler-simulator.sigs.k8s.io/score-result: >-
+      {"node-jjfg5":{"ImageLocality":"0","NodeAffinity":"0","NodeResourcesBalancedAllocation":"52","NodeResourcesFit":"47","TaintToleration":"0","VolumeBinding":"0"},"node-mtb5x":{"ImageLocality":"0","NodeAffinity":"0","NodeResourcesBalancedAllocation":"76","NodeResourcesFit":"73","TaintToleration":"0","VolumeBinding":"0"}}
+    kube-scheduler-simulator.sigs.k8s.io/selected-node: node-mtb5x
+```
+
+Users can also integrate their custom plugins or extenders, into the debuggable scheduler and visualize their results. 
+
+This debuggable scheduler can also run standalone, e.g., on any Kubernetes cluster or in integration tests. 
+This would be useful to custom plugin developers who want to test their plugins or examine their custom scheduler in a real cluster with better debuggability.
```

Comment:
Do you have an example that you can link? Is it actually used this way in some OSS projects that can be linked here?

Reply:
As a minumum, link some readme here

Reply:
I don't think any OSS project uses it, my previous company uses it internally though.